js/widgets/status.js

```diff
@@ -277,11 +277,11 @@ function StatusMatrix() {
                         }
                         break;
                     default:
-                        value = this._logo.blocks.blockList[this._logo.statusFields[i][0]].value;
+                        value = logo.blocks.blockList[logo.statusFields[i][0]].value;
                         break;
                 }
 
-                this._logo.inStatusMatrix = saveStatus;
+                logo.inStatusMatrix = saveStatus;
 
                 cell = this._statusTable.rows[i + 1].cells[activeTurtles + 1];
                 if (cell != null) {
```

Comment:
Please look for variables which are used before being defined. Ex - `note` variable in `updateAll` function. Please use `let` or `const` to define them before usage. You may lint the file using ES-Lint to find such issues, we enabled `no-undef` rule recently. Thanks